build_helpers/buildNPMInternals.sh

```diff
@@ -14,7 +14,7 @@ if (!fs.existsSync(internalPath)) {
 
 var providesModuleRegex = /@providesModule ([^\s*]+)/;
 var moduleRequireRegex = /=\s+require\((?:'|")([\w\.\/]+)(?:'|")\);/gm;
-var excludePathRegex = /^react($|\/)/;
+var excludePathRegex = /^(react|lodash|redux)($|\/)/;
```

Comment:
I don't think so, I'm guessing that's just how it was originally hacked put together.  One note is you'd need \/?.*$ since we want to handle a case like react/blah.  I can change if to use \/?.*$ if we'd like.